code/data_structures/src/list/circular_linked_list/circular_linked_list.c

```diff
@@ -0,0 +1,144 @@
+#include<stdio.h>
+#include<stdlib.h>
+typedef struct Node node;
+void display(node *tail);
+node *insert_at_end(node *tail);
+node *insert_at_front(node *tail);
+node *delete_at_begin(node *tail);
+node *delete_at_end(node *tail);
+
+struct Node{
+int Data;

```

Comment:
Fix indentation here.

Reply:
sorry,But where.In structure or in declaration section of functions.

Reply:
i have updated now.Please review it.